include/mxnet/ndarray.h

```diff
@@ -156,7 +156,7 @@ class NDArray {
   }
 
   /* \brief Check whether the two arrays are the same array */
-  inline bool IsSame(const NDArray& other) {
+  inline bool IsSame(const NDArray& other) const {
```

Comment:
@piiswrong I made the change here so that I can also call this function when I have a ``` const NDArray``` object.